src/environments/environment.ts

```diff
@@ -13,6 +13,7 @@ export const environment = {
   // createQuestionUri: `${baseUri}:${port}/questions/create`,
   responsesUri: `${baseUri}:${port}/responses`,
   userUri: `${baseUri}:${port}/users`,
+  userProfile: `${baseUri}:${port}/users/profile`,
```

Comment:
follow convention unless you have a good reason not to.  Does this endpoint exist?